### PR TITLE
Add example in create API for create index pattern, vega visualizatinn, dashboards for docs

### PR DESCRIPTION
**Pull request by zhyuanqi**
### Description

Add example in create API for create index pattern, vega visualizatinn, dashboards for docs

### Issues Resolved

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6719

## Screenshot

Screenshot 2024-05-29 at 11 08 40 AM
Screenshot 2024-05-29 at 11 08 46 AM
Screenshot 2024-05-29 at 11 08 52 AM

## Changelog
- doc: Add example for saved object creation part for openapi doc.

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
     - [Client_management_design](multi-datasource/client_management_design.md)
     - [High_level_design](multi-datasource/high_level_design.md)
     - [User_stories](multi-datasource/user_stories.md)
+  - [Openapi](openapi/README.md)
```
Comment (zhyuanqi) [inline docs/_sidebar.md:13]: This change is auto generated by the Docsify

File: docs/openapi/saved_objects/saved_objects.yml [hunk 2]
```diff
@@ -80,6 +80,50 @@ paths:
...
+                summary: Example of creating a index pattern saved object
+                value:
+                  attributes:
+                    title: 'my-index-pattern'
+                    fields: '[{"count":"1","name":"@timestamp","searchable":"true"}]'
+                  references:
+                    - id: '51339560-1d7c-11ef-b757-55fac6c80d9a'
+                      name: 'dataSource'
+                      type: 'data-source'
+              vegavVisualization:
```
Comment (BionIT) [inline docs/openapi/saved_objects/saved_objects.yml:94]: typo? 
```suggestion
              vegaVisualization:
```

File: docs/openapi/saved_objects/saved_objects.yml [hunk 3]
```diff
@@ -80,6 +80,50 @@ paths:
                   items:
                     type: string
                   description: Workspaces that this saved object exists in.
+            examples:
+              indexPattern:
+                summary: Example of creating a index pattern saved object
```
Comment (BionIT) [inline docs/openapi/saved_objects/saved_objects.yml:85]:
```suggestion
                summary: Example of creating an index pattern saved object
```

File: docs/_sidebar.md [hunk 4]
```diff
@@ -172,6 +174,7 @@
   - scripts
     - [README](../scripts/README.md)
   - [DOCS_README](DOCS_README.md)
+  - [Theme](theme.md)
```
Comment (yujin-emma) [inline docs/_sidebar.md:177]: is this also auto generated?

Comment (zhyuanqi) [reply docs/_sidebar.md:177]: Yes. It read through all the md file and updated the doc here.

**Comment by BionIT** [comment]
@joshuali925 could you take a look as well?

File: docs/openapi/saved_objects/saved_objects.yml [hunk 5]
```diff
@@ -80,6 +80,50 @@ paths:
...
+                      type: 'data-source'
+              vegaVisualization:
+                summary: Example of creating a Vega visualization saved object
+                value:
+                  attributes:
+                    title: 'my-vega-visualization'
+                    visState: '{"title":"vegaVisualization","type":"vega","aggs":[]}}'
+                    uiStateJSON: '{}'
+                    description: ''
+                    version: 1,
```
Comment (joshuali925) [inline docs/openapi/saved_objects/saved_objects.yml:102]: extra comma?

**Review by BionIT — APPROVED** [review]

**Review by zhongnansu — APPROVED** [review]

**Review by ZilongX — APPROVED** [review]

**Pull request merged**